### ISSUE TITLE
Slot blocker incompatible with empty/neutral airbases and FARPs

### ISSUE THREAD

**Issue by madrabbit711** [ModernColdWar/RedStormRising#156]
https://github.com/ModernColdWar/RedStormRising/blob/f1100abd394ebafea0c152deb32b7dcf07af47bf/slotBlocker.lua#L40-L52

slot blocker uses DCS side to determine if slots should be blocked and not the baseOwnership data.   However this is required for ship slot blocking.  As such, need to incorporate additional check for base status utilising `state.getOwner(baseName)`.

**Comment by madrabbit711** [comment]
Slot blocker only not compatible with gas platforms with no ground units, which has been recently fixed:
https://github.com/ModernColdWar/RedStormRising/blob/c72db143217cda8c3e90a99c75adbdf65de122da/slotBlocker.lua#L33-L38

The problem is not the slot blocker.  The problem is the baseOwnershipCheck relies on DCS to enumerate ground unit composition for a base/FARP which then feeds into the slot blocker.

In some recent cases, DCS has been reporting the base as neutral, despite being held by ground units from only one side. 
https://github.com/ModernColdWar/RedStormRising/issues/157

So the slot blocker only allows neutral coalition player to use those slots, of which there are none.

In short, we need our own unit enumeration in the future to feed correct base ownership information to the slot blocker:
https://github.com/ModernColdWar/RedStormRising/issues/159

Closing this issue.
